index.js

```diff
@@ -276,7 +280,8 @@ class McEncryptionSchemePolyfill {
         await McEncryptionSchemePolyfill.originalDecodingInfo_.call(
             this, requestedConfiguration);
 
-    if (!requestedConfiguration.keySystemConfiguration) {
+    if (!requestedConfiguration.keySystemConfiguration ||
+        capabilities.keySystemAccess) {
```

Comment:
This would have to be !capabilities.keySystemAccess.  Otherwise, the probe is skipped for any successful DRM content.